### PR TITLE
input address formatter

### PR DESCRIPTION
**Pull request by jdevcs**
## Description

Please include a summary of the changes and be sure to follow our [Contribution Guidelines](../CONTRIBUTIONS.md).

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] I have selected the correct base branch.
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [ ] My changes generate no new warnings.
- [ ] Any dependent changes have been merged and published in downstream modules.
- [ ] I ran `npm run dtslint` with success and extended the tests and types if necessary.
- [ ] I ran `npm run test:unit` with success.
- [ ] I ran `npm run test:cov` and my test cases cover all the lines and branches of the added code.
- [ ] I ran `npm run build` and tested `dist/web3.min.js` in a browser.
- [ ] I have tested my code on the live network.
- [ ] I have checked the Deploy Preview and it looks correct.
- [ ] I have updated the `CHANGELOG.md` file in the root folder.

### CODE REVIEW

File: packages/web3-eth-iban/test/fixtures/iban.ts [hunk 1]
```diff
@@ -8,9 +7,15 @@ export const validIbanToAddressData: [string, string][] = [
 	['XE76LL5FJYLSMDVW5J02HWU6R5ZVPHURYM9', '0xb8CE9ab6943e0eCED004cDe8e3bBed6568B2Fa01'],
 ];
 
-export const invalidIbanToAddressData: [string, IbanLengthError][] = [
-	['XE65GB6LDNXYOFTX0NSV3FUWKOWIXAMJK', new IbanLengthError()],
-	['XE65GB6LDNXYOFTX0NSV3FUWKOWIXAMJK365', new IbanLengthError()],
+export const invalidIbanToAddressData: [string, Error][] = [
+	[
+		'XE65GB6LDNXYOFTX0NSV3FUWKOWIXAMJK',
+		new Error('Iban is indirect and cannot be converted. Must be length of 34 or 35'),
```
Comment (jdevcs) [inline packages/web3-eth-iban/test/fixtures/iban.ts:13]: For Removing circular dependency https://github.com/ChainSafe/web3.js/pull/4742/commits/6b752ba74246d270a2b6e177b234e4ae69d9e5b2

File: packages/web3-eth-iban/src/index.ts [hunk 2]
```diff
@@ -79,7 +83,7 @@ export class Iban {
 			const paddedBigInt = leftPad(parsedBigInt, 40);
 			return toChecksumAddress(paddedBigInt);
 		}
-		throw new IbanLengthError();
+		throw new Error('Iban is indirect and cannot be converted. Must be length of 34 or 35');
```
Comment (jdevcs) [inline packages/web3-eth-iban/src/index.ts:86]: For Removing circular dependency https://github.com/ChainSafe/web3.js/pull/4742/commits/6b752ba74246d270a2b6e177b234e4ae69d9e5b2

File: packages/web3-eth-iban/package.json [hunk 3]
```diff
@@ -39,7 +39,6 @@
 		"typescript": "^4.5.2"
 	},
 	"dependencies": {
-		"web3-common": "1.0.0-alpha.0",
```
Comment (jdevcs) [inline packages/web3-eth-iban/package.json:42]: For Removing circular dependency https://github.com/ChainSafe/web3.js/pull/4742/commits/6b752ba74246d270a2b6e177b234e4ae69d9e5b2

File: packages/web3-common/src/errors.ts [hunk 4]
```diff
@@ -482,10 +482,3 @@ export class PBKDF2IterationsError extends Web3Error {
 		super('c > 1000, pbkdf2 is less secure with less iterations');
 	}
 }
-
```
Comment (jdevcs) [inline packages/web3-common/src/errors.ts:485]: For Removing circular dependency https://github.com/ChainSafe/web3.js/pull/4742/commits/6b752ba74246d270a2b6e177b234e4ae69d9e5b2

**Review by nazarhussain — APPROVED** [review]

**Review by luu-alex — APPROVED** [review]

**Pull request merged**